cinema/views.py

```diff
@@ -35,8 +45,14 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
+
+    def list(self, request, *args, **kwargs):
+        """Retrieve list of genres"""
+        return super().list(request, *args, **kwargs)
```

Comment:
The `list` method in `GenreViewSet` and other similar methods in this file are overriding the default behavior without adding any custom logic. Consider removing these methods if no additional functionality is needed.